node-authorizer/pkg/client/helper.go

```diff
@@ -98,7 +98,7 @@ func makeKubeconfig(ctx context.Context, config *Config, token string) ([]byte,
 			{
 				Name: clusterName,
 				Cluster: v1.Cluster{
-					Server: config.KubeAPI,
+					Server:                   config.KubeAPI,
```

Comment:
We verify using go 1.10 so this change is likely to fail. 1.11 is coming with later kops releases.